### ISSUE TITLE
Strange linebreaks in the speech

### ISSUE THREAD

**Issue by MansMeg** [swerik-project/riksdagen-records#50]
I was looking at this speech. It had really strange line breaks in the middle of the speech when using the swedeb interface.

{"id":"prot-1909--ak--024_026","gender":"Man","party":"S","year":1909,"speaker":"Karl Starbäck"}

Start of speech:
```
Herr Starbäck:
Herr talman! Jag hade verkligen icke tänkt yttra mig i den här frågan, ty jag hade trott, att det skulle bli ett meningsutbyte mellan juristerna här i kammaren om hvilken form för ändring af lagen eller hvilken form för. 
```

**Comment by BobBorges** [comment]
When reporting an error (in general, but especially from SWEDEB) we should get the ID attribute and/or line number of the xml element in question. It will make it easier to locate the problem example.

**Comment by MansMeg** [comment]
I agree! Ping @fredrik1984

**Comment by BobBorges** [comment]
Do we know how SWEDEB handles line breaks? It's possibly because of this:

![image](https://github.com/user-attachments/assets/d28202f1-d995-44a2-8e65-94c57b7c7341)

It's no problem in terms of TEI, but it also makes the thing difficult to render nicely. I think one of @ninpnin has talked about how to join these in a reasonable way. (right?)

**Comment by MansMeg** [comment]
Yes. I think this would be the reason. Ie its an example of segmentation error.

**Comment by fredrik1984** [comment]
Yes, when we in the Swerik project have implemented an ID for each speech, then SweDeb should use that (as well). I put this issue among the other segmentation-oriented issues in the backlog.
